### PR TITLE
DNS options update

### PR DESCRIPTION
**Pull request by afastronomer**
This PR is related to #4002 issue

### CODE REVIEW

**Review by lzdanski — CHANGES_REQUESTED** [review]
Let's get the AWS one ready and then I can move over the changes to make the others consistent - do these look ok?

File: astro/connect-aws.md [hunk 1]
```diff
@@ -401,6 +402,21 @@ When Astronomer support confirms that DNS forwarding was successfully implemente
 
 </TabItem>
 
+<TabItem value="Private hosted zone">
+
+Private hosted zones can be created for refleting particular DNS records in the customer environment. This solution doesn't require any changes or additional configurations at the customer side. Private zones work well when amount of zones and records is small and stable. Otherwise, name resolution accuracy and connectivity in general can be affected.
```
[the remark this answers is not in the document]
Comment (afastronomer) [reply astro/connect-aws.md:407]: Zones will be created on Astro side by CRE team.

File: astro/connect-aws.md [hunk 2]
```diff
@@ -401,6 +402,21 @@ When Astronomer support confirms that DNS forwarding was successfully implemente
...
+
+To use this solution, submit a request to [Astronomer support](https://cloud.astronomer.io/open-support-request). With your request, include the following information:
+
+- List of DNS records for the Private zone
```
[the remark this answers is not in the document]
Comment (afastronomer) [reply astro/connect-aws.md:411]: "Private" zone is a type name (another type is "Public"). That's why I used capitalization.

**Review by lzdanski — APPROVED** [review]

**Comment by afastronomer** [comment]
LGTM

**Pull request merged**